code/modules/reagents/reagents/food_drinks_vr.dm

```diff
@@ -536,6 +536,7 @@
 				H.feral = 0
 				to_chat(H, "<span class='info'>Your mind starts to clear, soothed into a state of clarity as your senses return.</span>")
 				log_and_message_admins("is no longer feral.", H)
+	..()
```

Comment:
This should be probably at the top of the proc, to be consistent. At roughly line 256?

Reply:
amended, also fixed the check above to be consistent with the recent feralness refactor